### ISSUE TITLE
Smooth out legacy wheel

### ISSUE THREAD

**Issue by 1nsertTextHere** [lukawarren/Lorien-Legacies#108]
I think you should add the individual keybinds back, but also have the "wheel" as an option as well.

**Comment by 1nsertTextHere** [comment]
or maybe just make it so you don't have to go all the way to the side of your screen to select the legacies, maybe just in the general direction like other action wheels.

**Comment by lukawarren** [comment]
Pretty sure you can just move your mouse towards the general direction and it will highlight whatever's closest, the code just has a minimum distance so you can get out of the wheel without being forced to do something. Did that stop working after the port to 1.16?

**Comment by 1nsertTextHere** [comment]
I think so, because i have to move my mouse over the name of the legacy to select it

**Comment by lukawarren** [comment]
Weird, it works for me after about half the distance from the centre. I assume that works for you? I could reduce the threshold.

**Comment by 1nsertTextHere** [comment]
yeah, I think that would be better

**Comment by 1nsertTextHere** [comment]
let me try again tho

**Comment by 1nsertTextHere** [comment]
maybe a little less, But i see it now, im just blind

**Comment by 1nsertTextHere** [comment]
actually, I think the thing that was tripping me up was the legacy names scrolling to the sides of the screen

**Comment by lukawarren** [comment]
The fancy animation? Yeah, they don't affect the minimum distance. I'll just halve its size.
